deploy/bundle/windows-machine-config-operator/manifests/windows-machine-config-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+  description: |
+    This operator allows for the management of Windows nodes in an OpenShift
+    cluster.
+  maturity: alpha
+  version: 0.0.1
+  replaces: ''
+  skips: []
+  minKubeVersion: ''
+  keywords: []
+  maintainers:
+    - name: ''
+      email: ''
+  provider:
+    name: Red Hat
+  labels: {}
+  selector:
+    matchLabels: {}
+  links:
+    - name: Repository
+      url: 'https://github.com/openshift/windows-machine-config-operator'
+#  icon:
+#    - base64data: ''
+#      mediatype: ''
+  customresourcedefinitions:
+    owned:
+      - name: windowsmachineconfigs.wmc.openshift.io
+        displayName: WindowsMachineConfig
+        kind: WindowsMachineConfig
+        version: v1alpha1
+        description: Windows Machine Config
+        resources:
+          - version: v1
+            kind: Deployment
+          - version: v1
+            kind: Service
+          - version: v1
+            kind: ReplicaSet
+          - version: v1
+            kind: Pod
+        specDescriptors: []
+        statusDescriptors: []
+    required: []
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+        - serviceAccountName: windows-machine-config-operator
+          rules:
+          - apiGroups:
+            - ""
+            resources:
+            - pods
+            - services
+            - services/finalizers
+            - endpoints
+            - persistentvolumeclaims
+            - events
+            - configmaps
+            - secrets
+            verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+          - apiGroups:
+            - apps
+            resources:
+            - deployments
+            - daemonsets
+            - replicasets
+            - statefulsets
+            verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+          - apiGroups:
+            - monitoring.coreos.com
+            resources:
+            - servicemonitors
+            verbs:
+            - get
+            - create
+          - apiGroups:
+            - apps
+            resourceNames:
+            - windows-machine-config-operator
+            resources:
+            - deployments/finalizers
+            verbs:
+            - update
+          - apiGroups:
+            - ""
+            resources:
+            - pods
+            verbs:
+            - get
+          - apiGroups:
+            - apps
+            resources:
+            - replicasets
+            - deployments
+            verbs:
+            - get
+          - apiGroups:
+            - wmc.openshift.io
+            resources:
+            - '*'
+            verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+      deployments:
```

Comment:
I know there was some talk about being use a separate YAML and referencing it. Please check with the OLM team.